src/streams/openssl.c

```diff
@@ -627,6 +627,16 @@ int git_openssl_stream_new(git_stream **out, const char *host, const char *port)
 	return error;
 }
 
+int git_openssl_set_cert_file(const char *file, const char *path)
```

Comment:
We usually use "__" to denote private API. In this case "git_openssl__set_cert_file".